src/main/java/org/usfirst/frc4904/auton/JustShoot.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.auton;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.Shoot;
+
+class JustShoot extends AutonRoutine {
+
+    public JustShoot() {
+        andThen(new Shoot(RobotMap.Component.indexer, RobotMap.Component.shooter, 0));
```

Comment:
Depending on how we end up calling shoot (ie whether or not it pulls the distance or we have to pass it in), this may change.

Reply:
Also, isn't this setting the speed to 0?

Reply:
Yes, zero was just meant as a placeholder before i knew the actual value to set it to, but i changed it to 0.5.